libhif/hif-context.c

```diff
@@ -997,8 +997,13 @@ hif_context_set_os_release(HifContext *context, GError **error)
     if (!g_file_get_contents(os_release, &contents, NULL, NULL)) {
         g_free (os_release);
         os_release = g_build_filename(source_root, "usr/lib/os-release", NULL);
-        if (!g_file_get_contents(os_release, &contents, NULL, NULL))
+        if (!g_file_get_contents(os_release, &contents, NULL, NULL)) {
+            g_set_error(error,
+                        HIF_ERROR,
+                        HIF_ERROR_FILE_NOT_FOUND,
```

Comment:
Not necessarily...we could get EIO or EPERM etc.  So wouldn't it be simpler to just pass `error` instead of `NULL` as the last arg to `g_file_get_contents()` ?